runtime/port/CMakeLists.txt

```diff
@@ -166,6 +166,10 @@ if(OMR_OS_LINUX)
 	target_link_libraries(j9prt PRIVATE rt)
 endif()
 
+if(OMR_OS_OSX)
+	target_link_libraries(j9prt PRIVATE iconv)
+endif()
```

Comment:
`module.xml` enables this for both OSX and AIX.  Can you add AIX into this?

https://github.com/eclipse/openj9/blob/e0da633168506fe61da53586e4bfd1689f3df08f/runtime/port/module.xml#L350-L353

Reply:
Updated